<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorLogWidgetClass</class>
 <widget class="QMainWindow" name="ErrorLogWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>568</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>运行错误信息</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1,0,1,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>错误类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_ErrType">
           <item>
            <property name="text">
             <string>所有</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>通用</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>机械臂</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>涂布机</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>光机</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>开始时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_StartTime">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="minimumDate">
            <date>
             <year>2023</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
           <property name="date">
            <date>
             <year>2023</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>结束时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_EndTime">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="minimumDate">
            <date>
             <year>2023</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
           <property name="date">
            <date>
             <year>2023</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ScreenErr">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>筛选</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_DeleteErr">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_ErrInfo">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <row/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
